src/Microsoft.Azure.Batch.SoftwareEntitlement.Client.Native/README.md

```diff
@@ -107,7 +114,7 @@ Microsoft::Azure::Batch::SoftwareEntitlement::Cleanup();
 ```
 
 ## Limitations
-The root certificate in the server's SSL certificate chain cannot be referenced by the ```ssl_cert_thumbprint```.
+The root certificate in the server's SSL certificate chain cannot be referenced by the ```ssl_cert_thumbprint``` specified in the call to ```AddSslCertificate```.
```

Comment:
Okay.  Perhaps rephrase the comment to clarify?  E.g. "When calling `AddSsslCertficate`, you must not specify the thumbprint of the root certificate in the server's SSL certificate chain. This is because OpenSSL does not include the root certificate in the list of certificates."